_data/projects/redux-saga-test-plan.yml

```diff
@@ -0,0 +1,15 @@
+name: Redux Saga Test Plan
+desc: Test Redux Saga with an easy plan! Declarative APIs for unit testing and integration testing sagas.
+site: https://github.com/jfairbank/redux-saga-test-plan
+tags:
+  - javascript
+  - redux-saga
+  - redux
+  - react
+  - sagas
+  - testing
+  - unit testing
+  - integration testing
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/jfairbank/redux-saga-test-plan/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```

Comment:
FYI, your link on L15 will not render an issue count on the website, as we're using a certain mechanism to parse this link for what to query the Github Issues API for. 

I'd recommend using this link: `https://github.com/jfairbank/redux-saga-test-plan/labels/up-for-grabs`, as that's the format we're expecting.